Integration can not be loaded

HA 2022.3.7
Integation no longer works.
From the log:
Felet härrör från en anpassad integration.

Logger: homeassistant.setup
Source: custom_components/automower/__init__.py:217
Integration: automower (documentation)
First occurred: 10:10:40 (1 occurrences)
Last logged: 10:10:40

Error during setup of component automower
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 235, in _async_setup_component
    result = await task
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/automower/__init__.py", line 217, in setup
    robots = api.list_robots()
  File "/usr/local/lib/python3.9/site-packages/pyhusmow/husmow.py", line 162, in list_robots
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 960, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://amc-api.dss.husqvarnagroup.net/v1/mowers

No dice with pyhusmow:

```
root@f66541666602:/# husmow --login login@gmail.com --password
Logger configured
Logged in successfully
Updated token
[ERROR] 404 Client Error: Not Found for url: https://amc-api.dss.husqvarnagroup.net/v1/mowers. Retrying to send the command 1
[ERROR] 404 Client Error: Not Found for url: https://amc-api.dss.husqvarnagroup.net/v1/mowers. Retrying to send the command 2
[ERROR] Failed to send the command
Done
```

--- a/custom_components/automower/__init__.py
+++ b/custom_components/automower/__init__.py
@@ -30,7 +30,6 @@ _LOGGER = logging.getLogger(__name__)
 
 DEFAULT_ICON = 'mdi:robot-mower'
 DOMAIN = 'automower'
-REQUIREMENTS = ['pyhusmow==0.1.1']
 VENDOR = 'Husqvarna'
 
 # TODO: Add more statuses as we observe them

--- a/custom_components/automower/manifest.json
+++ b/custom_components/automower/manifest.json
@@ -5,5 +5,5 @@
   "dependencies": [],
   "codeowners": ["@walthowd"],
   "version": "1.1.3",
-  "requirements": ["pyhusmow==0.1.1"]
+  "requirements": ["pyhusmow @ git+https://github.com/MAFLO321/pyhusmow.git@v0.3.0"]
 }

